R/lint.R

```diff
@@ -81,6 +81,13 @@ lint <- function(filename, linters = NULL, cache = FALSE, ..., parse_settings =
       else {
         expr_lints <- flatten_lints(linters[[linter]](expr)) # nolint
 
+        if (length(expr_lints)) {
+          expr_lints[] <- lapply(expr_lints, function(lint) {
```

Comment:
how about moving this in to `flatten_list` instead?

Reply:
Thought about this too, but all other uses of `flatten_lints` / `flatten_list` don't have a linter name as context, so we'd need a conditional there which will only be fulfilled in this single call from `lint()`.

Reply:
Got it